src/WindowsAzureMessaging/WindowsAzureMessaging/Models/MSNotificationHubMessage.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef MSNotificationHubMessage_h
+#define MSNotificationHubMessage_h
+
+#import <Foundation/Foundation.h>
+
+@interface MSNotificationHubMessage : NSObject
+
+/**
+ * Notification title.
+ */
+@property(nonatomic, copy, readonly) NSString *title;
+
+/**
+ * Notification message.
+ */
+@property(nonatomic, copy, readonly) NSString *message;
```

Comment:
APNS uses the term `body`. We shouldn't change it.

Reply:
Changed from message to body